generator/internal/descriptor_utils.cc

```diff
@@ -38,9 +38,23 @@
 #include <regex>
 #include <string>
 
+// #include <google/protobuf/compiler/cpp/cpp_helpers.h> doesn't work with the
```

Comment:
Seems to me that protobuf does not install that header at all, and therefore it is not intended for public  consumption.  I assume either they have a bug (they intend to install this, but just don't for some reason), or we are using an implementation detail.  May be worth checking with the protobuf folks about how they intend implementors of C++ plugins to use this?

Reply:
Changed to use our own keyword-mapping function, which can be enhanced as the need arises.  For now, just "namespace".

Reply:
It turns out that the function I really should have been using from `cpp_helpers.h` is also declared in `cpp_names.h`, which is public.  So, I'm going to close this and send a new PR using that instead.